daml-assistant/daml-project-config/DA/Daml/Project/Consts.hs

```diff
@@ -97,6 +103,7 @@ sdkConfigName = "sdk-config.yaml"
 damlEnvVars :: [String]
 damlEnvVars =
     [ damlPathEnvVar
+    , damlCacheEnvVar
```

Comment:
This is not going to work! The assistant filters out damlEnvVars that it doesn't set. This list should only contain variables that are "owned" by the assistant. See my comment above.